build_projects/shared-build-targets-utils/DependencyVersions.cs

```diff
@@ -7,9 +7,9 @@ namespace Microsoft.DotNet.Cli.Build
 {
     public class DependencyVersions
     {
-        public static readonly string CoreCLRVersion = "1.0.2";
-        public static readonly string JitVersion = "1.0.2";
-        public static readonly string CoreCLRVersion_Latest = "1.1.0-beta-24415-01";
-        public static readonly string JitVersion_Latest = "1.1.0-beta-24415-01";
+        public static readonly string CoreCLRVersion = "1.1.0-preview1-24508-06";
+        public static readonly string JitVersion = "1.1.0-preview1-24508-06";
+        public static readonly string CoreCLRVersion_Latest = "1.1.0-preview1-24508-06";
+        public static readonly string JitVersion_Latest = "1.1.0-preview1-24508-06";
```

Comment:
I added the _Latest properties so we could build win10-arm64 core-setup using the non-stable coreFX and CoreCLR dependencies. There wasn't a clean way to use the existing variables, so I made those new ones.

If `CoreCLRVersion` and `JitVersion` are going to be updated to the non-stable versions, then we can remove `CoreCLRVersion_Latest` and `JitVersion_Latest`  and replace all of their usage.